src/platform_impl/linux/event_loop.rs

```diff
@@ -567,7 +567,13 @@ impl<T: 'static> EventLoop<T> {
                 log::warn!("Failed to send window moved event to event channel: {}", e);
               }
 
-              let (w, h) = event.size();
+              let (w, h) = if let Some(child) = window.child() {
+                let allocation = child.allocation();
+                (allocation.width() as u32, allocation.height() as u32)
+              } else {
+                event.size()
+              };
+
```

Comment:
In `window.connect_configure_event`, `event.size()` represented the full window size. I have modified it to fetch the child size and display the inner size instead.

## Before the fix
<img width="1027" alt="스크린샷 2024-10-01 오후 4 26 58" src="https://github.com/user-attachments/assets/47ce8ff6-5490-46e3-98e1-cd185d7d4825">

## After the fix
<img width="1027" alt="스크린샷 2024-10-01 오후 4 27 41" src="https://github.com/user-attachments/assets/7734eb4e-7376-45af-8f21-8f60d384dcc9">